### ISSUE TITLE
Volume forecasting validation

### ISSUE THREAD

**Issue by yuki-harmonic** [yuki-harmonic/VisaForecasting#5]
To evaluate "how well the model fits data", we need to do model validation. 
Set training data and test data set. Run the model using the training data set. Compare the result with test set.
